website/docs/faq.md

```diff
@@ -1,3 +1,9 @@
+---
+id: faq
+title: FAQ
+type: how-to
```

Comment:
do we want to call this "how-to" or "recipe"? I'm fine with both

Reply:
good point, I'll switch all the `how-to`s to `recipe`